parquet-hadoop/src/main/java/org/apache/parquet/hadoop/rewrite/RewriteOptions.java

```diff
@@ -350,6 +436,28 @@ public Builder indexCacheStrategy(IndexCache.CacheStrategy cacheStrategy) {
       return this;
     }
 
+    /**
+     * Set a flag weather columns from join files need to overwrite columns from input files.
```

Comment:
`weather` -> `whether`

Reply:
What is the overwrite strategy? If the input file joins file A and joins file B. All those files have a column: col, which has the highest priority?

Reply:
Maybe we should doc it here.

Reply:
> What is the overwrite strategy? If the input file joins file A and joins file B. All those files have a column: col, which has the highest priority?

All inputFilesToJoin file are currently expected to have the same schema. Additionally there is a flag `overwriteInputWithJoinColumns` which if set to true will prioritize columns from  inputFilesToJoin instead of inputFiles, it is `false` by default so be default inputFiles columns will be preserved.